jwst/datamodels/schemas/psfmask.schema.yaml

```diff
@@ -16,3 +16,8 @@ allOf:
       default: 1.0
       ndim: 2
       datatype: float32
+    exposure:
```

Comment:
Instead of adding explicit definitions for this keyword, which would be yet another copy of definitions from other schemas, just add a reference to the existing "keyword_exptype.schema" file, as is already being done above for other keywords like filter and coronmsk.